### ISSUE TITLE
Item possession

### ISSUE THREAD

**Issue by lurr** [Zerthick/ProtectionPerms#6]
(Feature request)

Listen to inventory events, and if the player does not have permission to posses the item (with a node, such as `protectionperms.item.possess.<itemID>`, the item is removed from their inventory.
This would be absolutely great for modded servers, for items that bypass protectionperms entirely, such as Thermal Expansion's Florbs (which can't be disabled in config >:( ).

**Comment by Zerthick** [comment]
I agree, this would be a great addition. I don't believe this is currently possible in sponge, I have to wait for [ChangeInventoryEvent.Transfer](https://jd.spongepowered.org/6.0.0/org/spongepowered/api/event/item/inventory/ChangeInventoryEvent.Transfer.html) to be implemented. However, once this is the case I'll be sure to add this feature. 😄

**Comment by Zerthick** [comment]
So I can implement it now, but it destroys the item that you are trying to pick up 🙁 looking to see if I can come up with a fix

**Comment by williamhatcher** [comment]
I don't know the Sponge API, but before the item is destroyed, could you copy the item and drop it where it was?

**Comment by Zerthick** [comment]
So you can now prevent players from picking up items from the ground, however in theory they can still get items by taking them out of chests, I'll need additional permission nodes to handle those other cases.

**Comment by williamhatcher** [comment]
I can confirm that players can place items into their inventory from chests, enderchests, etc. if they have the permission to use the container.

Maybe a `hold` or `in_inventory` node would be useful? 
If and admin did `/give <player> <restricted_item>`, the item would never go into the player's inventory.
